### PR TITLE
Fix issue #147, add utm params to external links

### PR DESCRIPTION
**Pull request by schalkneethling**
@chrismore r?

### CODE REVIEW

File: data/tmpl/values/content5-snippet.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <section id="allaboard_snippet" class="allaboard-snippet">
   <p>You’re different, that’s why we like you. Keep up with issues surrounding the Web.</p>
-  <a href="https://www.mozilla.org/#newsletter-subscribe" id="allaboard_anippet_intent" class="button">Stay informed</a>
+  <a href="https://www.mozilla.org/#newsletter-subscribe?utm_source=fxonboarding&amp;utm_medium=snippet&amp;utm_campaign=onboardingv1&amp;utm_content=community" id="allaboard_anippet_intent" class="button">Stay informed</a>
```
Comment (chrismore) [inline data/tmpl/values/content5-snippet.html]: @schalkneethling the bookmark in the URL needs to be after the query parameters. like:

https://www.mozilla.org/en-US/?utm_source=fxonboarding&utm_medium=snippet&utm_campaign=onboardingv1&utm_content=community#newsletter-subscribe

File: data/tmpl/values/content5.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
         </ul>
         <button id="tell_me" class="button">Show Answer</button>
         <div id="secondary_content" class="secondary-content hide">
-          <p>Love the Web? Get the <a href="https://www.mozilla.org/#newsletter-subscribe" id="secondary_cta" data-intent="newsletter">Mozilla newsletter</a> and help us keep it open and free.</p>
+          <p>Love the Web? Get the <a href="https://www.mozilla.org/#newsletter-subscribe?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=community" id="secondary_cta" data-intent="newsletter">Mozilla newsletter</a> and help us keep it open and free.</p>
```
Comment (chrismore) [inline data/tmpl/values/content5.html]: @schalkneethling same as above, reverse the order of the URL. query parameters first and then @newsletter-subscribe

File: data/tmpl/values/content5-snippet.html (hunk 1 above)
Comment (schalkneethling) [inline data/tmpl/values/content5-snippet.html]: Whoops, good catch

**Comment by schalkneethling** [comment]
@chrismore updated

**Comment by chrismore** [comment]
r+

**Pull request merged**